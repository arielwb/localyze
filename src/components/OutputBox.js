import React from 'react'

export const OutputBox = ({ children }) => {
  return (
    <pre>{children}</pre>
  )
}
