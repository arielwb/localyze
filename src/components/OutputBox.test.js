import React from 'react'
import renderer from 'react-test-renderer'
import { OutputBox } from './OutputBox'

describe('OutputBox', () => {
  it('should match the snapshot', () => {
    const tree = renderer
      .create(<OutputBox>A beautiful output</OutputBox>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
