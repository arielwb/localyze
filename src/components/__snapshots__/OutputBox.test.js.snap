// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OutputBox should match the snapshot 1`] = `
<pre>
  A beautiful output
</pre>
`;
